perforator/deploy/kubernetes/helm/releases.md

```diff
@@ -6,6 +6,7 @@ Enhancements:
 
 Internal changes:
 + Change auto-generated CA subject
++ Fix mount extra volumes
```

Comment:
Please, move this to a new "Fixes" list, above enhancements.

Otherwise looks great to me